release_notes/index.adoc

```diff
@@ -13,6 +13,9 @@ include::topics/templates/document-attributes-community.adoc[]
 :release_header_latest_link: {releasenotes_link_latest}
 include::topics/templates/release-header.adoc[]
 
+== {project_name_full} 14.0.0
```

Comment:
Should this be 
```suggestion
== {project_name_full} 13.0.0
```
?

Reply:
Well, I don't know, I thought 13.0.0 would be released at the time of artifact binding merge. Do you think we will manage to merge this before releasing Keycloak 13? If yes I will change.

Reply:
Yes, I believe so